byron-proxy/src/exec/ParseGenesis.hs

```diff
@@ -0,0 +1,11 @@
+import qualified Data.ByteString.Lazy as LBS
+import           System.IO (stdin)
+import           Text.JSON.Canonical (parseCanonicalJSON, renderCanonicalJSON)
+
+main :: IO ()
+main = do
+  rawData <- LBS.hGetContents stdin
+  let
+    Right value = parseCanonicalJSON rawData
+    cleaned = renderCanonicalJSON value
+  LBS.putStr cleaned
```

Comment:
Didn't we conclude we didn't need this utility? The node has switched back to allowing pretty input and doing the canonical hashing itself. Perhaps that's not on this branch yet.

Reply:
forgot to remove :) thanks for the catch!

Reply:
pushed validator-nix-with-json-parser branch as it might be useful for the genesis-hash tool we discussed earlier, but should be a separate tool.